packages/generate/src/syntax/toEdgeQL.ts

```diff
@@ -1454,7 +1454,21 @@ function literalToEdgeQL(type: BaseType, val: any): string {
   let stringRep;
   if (typename === "std::json") {
     skipCast = true;
-    stringRep = `to_json($$${JSON.stringify(val)}$$)`;
+    const stringified = JSON.stringify(val);
+    const buildLabel = (
+      dataStr: string,
+      currentLabel: string = "jsonliteral",
+    ) => {
+      if (dataStr.includes(`$${currentLabel}$`)) {
```

Comment:
can you replace this with while?

Reply:
and just error out if we fail to find a unique quote string after a few iterations

Reply:
i'd also replace random chars with just a sequence of random digits to make it less lucrative for someone to guess or try to anything funny